docs-markdown/src/test/suite/controllers/table-controller.test.ts

```diff
@@ -27,6 +27,9 @@ suite('Table Controller', () => {
 	teardown(() => {
 		chai.spy.restore(common);
 	});
+	suiteSetup(() => {
```

Comment:
It looks like this suiteSetup is already being done on line 23, and is causing this test to fail. Please remove this sendTelemetryData.

Reply:
fixed